css/comp.css

```diff
@@ -99,6 +99,7 @@
 
 .slider {
   width: 100%;
+}
 /* ----------------------------------------- Simplified Grid */
 
 .row-ex {
```

Comment:
Write proper name for class. It enhances code readability.

Reply:
sure. i'll keep that in mind.